.github/workflows/ci.yaml

```diff
@@ -34,4 +34,6 @@ jobs:
           echo NEXT_PUBLIC_CONVEX_URL=${{ secrets.NEXT_PUBLIC_CONVEX_URL }} >> .env.local
           npm run build
       # - name: Run test with coverage
-      #   run: pnpm run coverage
\ No newline at end of file
+      #   run: pnpm run coverage
```

Comment:
We're not doing the coverage yet, but I think we can implement it in a separate PR.

Reply:
Alright, changes have been made.